python/grass/jupyter/baseseriesmap.py

```diff
@@ -1,3 +1,15 @@
+#
+# AUTHOR(S): Riya Saxena <29riyasaxena AT gmail>
+#
+# PURPOSE:   This module contains functions for interactive visualizations
+#            in Jupyter Notebooks.
```

Comment:
This could be more about baseseriesmap, no?

Reply:
Yes, I have modified it.